purchase_invoice_indicator/models/account_move.py

```diff
@@ -4,12 +4,12 @@
 from odoo import api, fields, models
 
 
-class AccountInvoice(models.Model):
-    _inherit = "account.invoice"
+class AccountMove(models.Model):
+    _inherit = "account.move"
 
-    is_invoice = fields.Boolean("Is Invoice", readonly=True, store=True)
+    is_invoice_issuer = fields.Boolean(string="Is Invoice", readonly=True, store=True)
```

Comment:
Why you change field name?

Reply:
In odoo15, there is `is_invoice` method. So, if we use is_invoice field and call this function, odoo call field and get error.https://github.com/odoo/odoo/blob/2c6d0d7526ca75ec78d92e10e73b53330a42cb4f/addons/account/models/account_move.py#L544